src/app/modules/group/components/user-list/user-list.component.ts

```diff
@@ -19,6 +20,8 @@ export class UserListComponent implements OnChanges {
 
   members: Member[] = [];
 
+  @ViewChild('grid', { static: true }) private grid?: GridComponent;
```

Comment:
is there a good reason to make it static? If not, it should not be.

Reply:
fixed